css/components.css

```
@@ -20,50 +20,267 @@
+  font-weight: 700;
+  color: var(--primary-color);
+  letter-spacing: 0.5px;
+  margin-bottom: var(--number-margin);
+  line-height: 1.1;
+
+  /* Text overflow prevention */
+  word-wrap: break-word;
+  overflow-wrap: break-word;
+
+  /* Responsive font sizing */
+  font-size: clamp(1.5rem, 4vw, var(--number-size));
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

font-size is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)